### PR TITLE
Try to fix #41

### LINKED ISSUE #41: Travis Builds are Broken
This is a pain.

wampy uses eventlet under the hood for non-blocking IO. Travis recently move to 2.7.13 from 2.7.10. There is a bug that requires eventlet < 0.21.0 after making this move.
The issue is here: https://github.com/eventlet/eventlet/issues/401

But wampy requires eventlet >= 0.21.0 to avoid an SSL recursion bug in 3.6
The issue is here: https://github.com/eventlet/eventlet/issues/371

Options that I see are:
- abandon Python 2 entirely
- abandon Travis
- wait for one of the bugs to be fixed
- accept broken builds
- ask for hellp

I am now asking for help.....

Thanks

### PR DESCRIPTION
**Pull request by titilambert**

### CODE REVIEW

**Comment by titilambert** [comment]
@noisyboiler IT'S GREEN !!!!

File: .travis.yml [hunk 1]
```diff
@@ -10,5 +10,7 @@ python:
   - "3.6"
 
 script:
+  - pip install --upgrade setuptools
```
Comment (titilambert) [inline .travis.yml]: This fix tests for python 3,3

File: wampy/testing/pytest_plugin.py [hunk 2]
```diff
@@ -123,7 +123,7 @@ def find_processes(process_name):
 
 
 def get_process_ids():
-    output = find_processes("crossbar")
+    output = find_processes("crossbar-controller")
```
Comment (titilambert) [inline wampy/testing/pytest_plugin.py]: Improve crossbar shutdown

**Comment by noisyboiler** [comment]
thanks @titilambert !!!!

**Comment by noisyboiler** [comment]
really pleased about this. thanks again.

**Pull request merged**